recipe/meta.yaml

```diff
@@ -13,30 +13,30 @@ build:
   script: python setup.py install
 
 requirements:
+  host:
+    - python
+    - pip
+
   build:
-    - setuptools
     - python
-    - numpy
-    - scipy
-    - pytest
-    - pytest-runner
+    - pip
+    - setuptools
     - cmake
-    - pandas
-    - lmfit-c
-    - pybind11
     - gxx_linux-64
     - cpplint
+    - pybind11
 
   run:
-    - lmfit-c
-    - pandas
     - python
-    - {{ pin_compatible('numpy') }}
```

Comment:
I think this still needs to be here. I remember we struggled with this configuration a lot while developing this recipe in the lightsource2-recipes repo.